server/sonar-server/src/main/java/org/sonar/server/user/index/UserResultSetIterator.java

```diff
@@ -57,38 +60,38 @@
 
   private final ListMultimap<String, String> organizationUuidsByLogins;
 
-  private UserResultSetIterator(PreparedStatement stmt, ListMultimap<String, String> organizationUuidsByLogins) throws SQLException {
+  private UserResultSetIterator(Collection<EsQueueDto> esQueueDtos, PreparedStatement stmt, ListMultimap<String, String> organizationUuidsByLogins) throws SQLException {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "esQueueDtos". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1172)